content/guides/destructuring.adoc

```diff
@@ -13,6 +13,8 @@ toc::[]
 
 Destructuring is a way to concisely bind names to the values inside a data structure. Destructuring allows us to write more concise and readable code.
 
+Destructuring can appear in explicit or implicit `let` expressions. No special syntax is needed. Standard Clojure collection syntax is used on the left-hand side to specify a structure onto which a right-hand value is to be mapped.
```

Comment:
Destructuring can appear in these places, but also in an open-ended set of other places (some that are not explicit or implicit let expressions, like defn params). I think it would be useful to say something about this, but as it is, is misleading.

Special syntax *is* needed - the destructuring  dsl is what this page is all about. So I would remove the last 2 sentences.

Reply:
Regarding "special syntax" I was thinking about about a new kind of "syntax tree node" to introduce destructuring. For example < [ x y ] [ 1 2 ] > which would tell the developer that "in between the <>, destructuring occurs". But that's not what we have just the syntax for a two-element vector [ [ x y ] [ 1 2 ] ], plus the context. Very uniform.

Reply:
It's Clojure data structures, like all other syntax. But it is special in structure, please remove.